db/migrate/20230922232149_change_author_id_to_user_id_in_posts.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAuthorIdToUserIdInPosts < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :posts, :author_id, :user_id
+  end
```

Comment:
- [ ] You should delete this migration, you are deleting the author_id to the user while it is what is required in the requirement, your migration should look like this
` reference author classne{ user}`-- >> something simullar to this

Reply:
Solved :+1: